_specs/ecip-1104.md

```diff
@@ -44,16 +42,18 @@ The fee market change and bomb delay are omitted at this time as they are not ap
 ##### Omitted
 - EIP 1559 "Fee market change": A transaction pricing mechanism that includes fixed-per-block network fee that is burned and dynamically expands/contracts block sizes to deal with transient congestion.
   - Undesirable to implement at this time as it would conflict with the current monetary policy set in ECIP-1017  
+- EIP 3198 "BASEFEE opcode": Adds an opcode that gives the EVM access to the block’s base fee. 
+  - It depends on EIP-1559 and it wouldn't make sense in the context of ETC
```

Comment:
```suggestion
  - It depends on EIP-1559. Behavior specifications for use in context without EIP-1559 are undefined.
```

Reply:
Thanks for the suggestion! I already applied it